src/index.html

```diff
@@ -2,15 +2,45 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="#"
```

Comment:
```suggestion
      action=""
```

Reply:
![lint-err](https://user-images.githubusercontent.com/98541281/204094445-02105803-73ad-4bbd-81fe-abba67d2c8e1.png)
I have a problem with linter after removing here the "#", it shows the error and doesn't let me to move forward.